bigbluebutton-html5/imports/ui/components/presentation/component.jsx

```diff
@@ -258,10 +264,13 @@ class Presentation extends PureComponent {
     const presentationSizes = this.getPresentationSizesAvailable();
     if (Object.keys(presentationSizes).length > 0) {
       // updating the size of the space available for the slide
-      this.setState({
-        presentationHeight: presentationSizes.presentationHeight,
-        presentationWidth: presentationSizes.presentationWidth,
-      });
+      const { isUnmounted } = this.state;
```

Comment:
Instead of using a state variable, consider using Session package and setting a variable on it.

And here, you can access it by calling `Session.get()`:
`const isUnmounted = Session.get('presentationComponentUnmounted');`